packages/cache/src/internal/cacheUtils.ts

```diff
@@ -94,18 +96,14 @@ async function getVersion(app: string): Promise<string> {
 
 // Use zstandard if possible to maximize cache performance
 export async function getCompressionMethod(): Promise<CompressionMethod> {
-  const versionOutput = await getVersion('zstd')
+  const versionOutput = await getVersion('zstd', ['--quiet'])
   const version = semver.clean(versionOutput)
 
-  if (!versionOutput.toLowerCase().includes('zstd command line interface')) {
+  if (versionOutput === '' || version === null) {
```

Comment:
Yes Bishal, I took a call to remove that from PR as it was currently redundant and we were not using `--long` at all. Since folks are having issues using `zstd` currently, I was thinking we can introduce `--long` after testing that it works but release now to unblock `zstd` usage. Although the change to fix `long` is also not much work.

Reply:
Ok.